### DISCUSSION TITLE
Organization accounts for Visual Studio Professional with GitHub Enterprise

### DISCUSSION THREAD

**Discussion by HatemRafrafi** [community/community#110921] [Enterprise]
i want to know how can i connect to github entreprise with my license Visual Studio Professional with GitHub Enterprise and i know our subscription linked to any organisation. 
how can i create organisation linked to our contrat in visual subscription

**Comment by lijoneel** [comment]
Having a similar issue where we have Visual Studio Professional with GitHub Enterprise subscriptions that are not applied to our GitHub accounts. @HatemRafrafi did you ever get a resolution?

**Comment by fsilva5** [comment]
I'm also having this problem @HatemRafrafi @lijoneel did you ever get a resolution? I opened a ticket, but they haven't been able to resolve it yet.
